### ISSUE TITLE
 find-tppcertificate -CreatedAfter returns all certs, not the after date.

### ISSUE THREAD

**Issue by doyle043** [gdbarron/VenafiTppPS#117]
find-tppcertificate -createdafter "2020-10-1" returns all certs in the system not the 5 that should be in that range

# Environment
v19.3.2
```
Operating System: Windows 10
VenafiTppPS version:  1.0.0      VenafiTppPS                         PSGallery
PowerShell version: PSVersion                      5.1.18362.1110
```

# Steps to reproduce

# Expected behavior

# Actual behavior

# Screenshots

**Comment by doyle043** [comment]
Had 1.0 in my profile, uninstalled, reinstalled 2.0.4 and see same behavior.

**Comment by doyle043** [comment]
Also doesn't matter whether date has leading zero's or not
find-tppcertificate -createdafter "2020-10-07"

**Comment by gdbarron** [comment]
Hey there @doyle043.  Can you please post a screenshot of the command with `-verbose`?

**Comment by gdbarron** [comment]
Nevermind.  I see that some filters were never implemented, not sure how they got missed.  I'll look to add those in.

**Comment by gdbarron** [comment]
@doyle043, resolved in v2.0.5.  #118
